internal/plugincommon/discovery/processor.go

```diff
@@ -147,10 +150,18 @@ func (cdp *ContractDiscoveryProcessor) Outcome(
 			cdp.lggr.Warnf("skipping empty nonce manager address in observation from Oracle %d", ao.OracleID)
 			continue
 		}
+		if len(ao.Observation.RMNRemote) == 0 {
+			cdp.lggr.Warnf("skipping empty RMNRemote address in observation from Oracle %d", ao.OracleID)
+			continue
+		}
```

Comment:
This will end up skipping a nonce manager that is valid, put this right before the rmnRemoteAddrs append